debian/changelog

```diff
@@ -1,97 +1,39 @@
-plainbox-provider-checkbox (0.59.0~ppa~ubuntu21.10.1) impish; urgency=low
+checkbox-provider-base (3.3.0~dev19~ubuntu24.04.1) noble; urgency=low
```

Comment:
You either need to:

1. Keep the old source package name
2. Change the name of the git repo to match the source package name
3. Change the jenkins job so it specifies the [source package name to use](https://github.com/endlessm/eos-build/blob/master/ci-integration/jenkins-jobs/obs.yaml#L260)

Reply:
Covered by https://github.com/endlessm/eos-build/pull/1540